src/player.cpp

```diff
@@ -3202,8 +3205,8 @@ void player::search_surroundings()
         }
         // Chance to detect traps we haven't yet seen.
         if( tr.detect_trap( tp, *this ) ) {
-            if( tr.get_visibility() > 0 ) {
-                // Only bug player about traps that aren't trivial to spot.
+            if( tr.get_visibility() >
+                0 ) {                // Only bug player about traps that aren't trivial to spot.
```

Comment:
looks like some more rogue whitespace here.

Reply:
Gah, astyle must really hate me. Anyone else ever get this?